src/DataProtection/Abstractions/test/DataProtectionCommonExtensionsTests.cs

```diff
@@ -249,7 +249,8 @@ public void Protect_InvalidUtf8_Failure()
         // Act & assert
         var ex = Assert.Throws<CryptographicException>(() =>
         {
-            mockProtector.Object.Protect("Hello\ud800");
+            string plainText = "Hello\ud800";
```

Comment:
Why not `var`❔ And, why is a variable necessary here at all❔

Reply:
Ah yeah, `var` works, wasn't sure if it would.

The variable is needed because when the string is used inline it tries to convert the string to a `byte[]` and use the `byte[]` overload of the method. In this specific case, converting to a utf8 `byte[]` won't work because of `\ud800`, and in other cases we actually want to call the string overload because it does base64 conversions.